client/signup/steps/import-from/squarespace/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { translate, TranslateOptions, TranslateOptionsText } from 'i18n-calypso';
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import importerConfig from 'calypso/lib/importer/importer-config';
+import { resetImport, startImport } from 'calypso/state/imports/actions';
+import { appStates } from 'calypso/state/imports/constants';
+import { importSite } from 'calypso/state/imports/site-importer/actions';
+import CompleteScreen from '../components/complete-screen';
+import GettingStartedVideo from '../components/getting-started-video';
+import ImporterDrag from '../components/importer-drag';
+import ProgressScreen from '../components/progress-screen';
+import { Importer, ImporterBaseProps, ImportJob, ImportJobParams } from '../types';
+import { getImporterTypeForEngine } from '../util';
+
+export const SquarespaceImporter: React.FunctionComponent< ImporterBaseProps > = ( props ) => {
+	const importer: Importer = 'squarespace';
+	const {
+		job,
+		urlData,
+		siteId,
+		site,
+		siteSlug,
+		fromSite,
+		importSite,
+		startImport,
+		resetImport,
+	} = props;
+	const importerData = importerConfig().squarespace;
+
+	populateMessages();
+
+	/**
+	 * Effects
+	 */
+	useEffect( runImport, [ job ] );
+
+	/**
+	 ↓ Methods
+	 */
+	function runImport() {
+		// If there is no existing import job, start a new
+		if ( job === undefined ) {
+			startImport( siteId, getImporterTypeForEngine( importer ) );
+		} else if ( job.importerState === appStates.READY_FOR_UPLOAD ) {
+			importSite( prepareImportParams() );
+		}
+	}
+
+	function prepareImportParams(): ImportJobParams {
+		const targetSiteUrl = fromSite.startsWith( 'http' ) ? fromSite : 'https://' + fromSite;
+
+		return {
+			engine: importer,
+			importerStatus: job as ImportJob,
+			params: { engine: importer },
+			site: { ID: siteId },
+			targetSiteUrl,
+			supportedContent: [],
+			unsupportedContent: [],
+		};
+	}
+
+	function checkProgress() {
+		return job?.importerState === appStates.IMPORTING;
+	}
+
+	function checkIsSuccess() {
+		return job?.importerState === appStates.IMPORT_SUCCESS;
+	}
+
+	function showVideoComponent() {
+		return checkProgress() || checkIsSuccess();
+	}
+
+	// Change the default messages
+	function populateMessages() {
+		const options: TranslateOptions = {
+			args: {
+				importerName: 'Squarespace',
+			},
+			components: {
+				supportLink: (
+					<InlineSupportLink supportContext="importers-squarespace" showIcon={ false }>
+						{ translate( 'Need help exporting your content?' ) }
+					</InlineSupportLink>
+				),
+			},
+		};
+
+		importerData.title = translate( 'Import content from %(importerName)s', {
+			...options,
+			textOnly: true,
+		} as TranslateOptionsText ) as string;
+
+		importerData.description = translate(
+			'Import your posts, pages, comments, tags, and images from your %(importerName)s export file.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Import posts, pages, comments, tags, and images from a %(importerName)s export file.' )` *ES Score: 7*

Reply:
@vishnugopal @a8ci18n This is is only slightly different from the default one. It only has a "your" word added. Maybe we should not have a new one.

Reply:
Agreed, let's replace!